configure.ac

```diff
@@ -89,13 +89,6 @@ esac
 
 AX_LUA_HEADERS
 AX_LUA_LIBS
-# Check for Lua modules
-AX_LUA_MODULE([lpeg], [lpeg])
```

Comment:
I'm not sure about this. I don't like the idea that someone can run ./configure && make and get a non-working product at the end. Why not leave these checks in and people can at least get told if they don't have the dependencies installed?

Reply:
Because `pkg-config` does not understand where to find the packages. We're doing a lot of manipulation to the lua path including re-exporting it with `luarocks`. At least in my cursory testing was that it was finding the system libraries, but not our manually supplied ones. That is, after all, what the tool is supposed to do: find stuff on the system not lint the current project. We don't need it to find the lua modules because we do that ourselves as part of the next step: `make`.